ExtractorUtils/Configuration/BaseConfig.cs

```diff
@@ -310,5 +310,26 @@ public class CertificateConfig
         public IEnumerable<string>? AllowList { get; set; }
     }
 
+    /// <summary>
+    /// Config for a RawExtractorManager.
+    /// </summary>
+    public class RawManagerConfig
```

Comment:
The whole "Manager" name is alright internally, but it does not make sense outside of that specific context. Looking at this I would have no clue what an "Extractor manager" is, certainly my first guess would _not_ be high availability.